gcc/rust/backend/rust-compile-intrinsic.cc

```diff
@@ -213,6 +214,9 @@ finalize_intrinsic_block (Context *ctx, tree fndecl)
   DECL_SAVED_TREE (fndecl) = bind_tree;
 
   ctx->push_function (fndecl);
+
+  DECL_DECLARED_CONSTEXPR_P (fndecl);
```

Comment:
Same here

Reply:
I think you are correct about the first one as that seems to be dealing with compiling a constant item, not sure about this one as this one gets called from quite a few places. I think @philberty would know better as I have never touched this file :)

Reply:
Yeah Arthur is right here it should be =1 this is my mistake.